### PR TITLE
Position based Actions and Changes to other actions

### PR DESCRIPTION
**Pull request by BBScholar**
I cannot test our actions without running the robot because the `Timer` class doesn't work without a robot.

### CODE REVIEW

**Comment by connorworley** [comment]
Can you mock the timer class?

**Comment by BBScholar** [comment]
We're not on that branch. And that branch throws too man errors

**Comment by BBScholar** [comment]
Should be gucci. Tested everything and it works

File: src/main/kotlin/frc/team5499/frc2018Kotlin/auto/actions/ParallelAction.kt [hunk 1]
```diff
@@ -24,9 +24,10 @@ class ParallelAction(timeout: Double, vararg actions: Action) : Action(timeout)
 
     @Suppress("ReturnCount")
     override fun next(): Boolean {
-        if (super.timedOut()) {
-            return true
-        }
+        // we dont want this to time out
```
Comment (FRCTeam5499) [inline src/main/kotlin/frc/team5499/frc2018Kotlin/auto/actions/ParallelAction.kt]: don't commit comments, they're saved in the git history forever anyways

**Pull request merged**